Fix README for CentOS images.

jerboaa opened the pull request:
PR https://github.com/redhat-developer/s2i-dotnetcore/pull/137 missed this
as @tmds pointed out. This should fix it.

jerboaa commented:
[skipci]

tmds commented on README.md:
```diff
@@ -46,12 +46,12 @@ Image name structure
 
 ##### Structure: 1/2-3-4
```
fyi, before I added centos prefix, it said: dotnet/1-2-3

suggestion: we can get rid of the numbers, and include the runtime: `dotnet/<prefix>-<version>-[runtime-]<os>`

tmds replied:
I don't have a strong preference, so feel free to keep it as-is.

jerboaa commented:
This seems simpler, so I've updated it. Removed the `builder` in section "Base image". It's really there to stress the base stack, not whether it's a s2i builder or not.

jerboaa commented:
@tmds Good to go?

tmds commented:
:shipit:

jerboaa commented:
Thanks for the review!
